### PR TITLE
Remove grpc imports from init and add pb package

### PR DESCRIPTION
**Pull request by smacker**
Superseded #39

The first commit has the same change but correct commit message with the explanation why we need it.
The second re-export grpc stuff as we agreed in a comment to that PR.

### CODE REVIEW

File: python/lookout/sdk/pb.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+"""
+Re-exporting all grpc related classes/functions for cleaner API
+"""
+
+from .service_analyzer_pb2_grpc import \
+    AnalyzerServicer, add_AnalyzerServicer_to_server as add_analyzer_to_server
+
+from .event_pb2_grpc import *
+from .event_pb2 import *
+from .grpc import *
```
Comment (vmarkovtsev) [inline python/lookout/sdk/pb.py]: Should we separate our code from autogenerated one? I would rather explicitly import this:

```python
from lookout.sdk.pb import AnalyzerServicer
from lookout.sdk.grpc import to_grpc_address
```

Comment (smacker) [reply python/lookout/sdk/pb.py]: done in https://github.com/src-d/lookout-sdk/pull/43/commits/03c416667581585f56c5412fc2348a95ef0aa722

File: python/lookout/sdk/pb.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+"""
+Re-exporting all grpc related classes/functions for cleaner API
+"""
+
+from .service_analyzer_pb2_grpc import \
+    AnalyzerServicer, add_AnalyzerServicer_to_server as add_analyzer_to_server
```
Comment (vmarkovtsev) [inline python/lookout/sdk/pb.py]: `AnalyzerServicer ` does not need to be imported here. Sorry - I should have spotted this earlier!

Comment (smacker) [reply python/lookout/sdk/pb.py]: True. Leftover after copy-past. Thank you!

**Pull request merged**